include/extractor/edge_based_graph_factory.hpp

```diff
@@ -42,6 +42,33 @@ namespace osrm
 namespace extractor
 {
 
+namespace lookup
+{
+// Set to 1 byte alignment
+#pragma pack(push, r1, 1)
```

Comment:
If there is a non-pragma approach I would prefer that. Also static_assert on size is important to document platform specific breakages.